sale_promotion_rule_display_discount_amount/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
```

Comment:
Could you add some missing entries and remove void ones ?

https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py